arch/armv7m/jmp.S

```diff
@@ -23,7 +23,7 @@
 .type _setjmp, %function
 _setjmp:
 	/* set jump address and stack pointer */
-	addw r1, pc, #((1f - . - 4) | 1) /* 4 is the size of addw instruction itself */
+	ldr r1, =(1f + 1)
```

Comment:
Can we do `ldr r1, =(1f | 1)` instead? I'm not sure if we can, but it would be more reliable (if for some reason should 1f have thumb bit set already)

Reply:
We can't, syntax error. Docs says that `ldr r1, =1f` should be sufficient in Thumb mode but when I disassemble it doesn't have last bit set.